server/src/termdb.barchart.js

```diff
@@ -634,9 +634,20 @@ Output: The function has no return but appends the statistical results to the pr
 	]
 }
 */
-async function computePvalues(data, ds) {
+async function computePvalues(data, ds, hiddenValues) {
+	// remove hidden values from deep copy of data
+	const charts = JSON.parse(JSON.stringify(data.charts))
```

Comment:
You should just create a new object from scratch called `charts`. You can then iterate through `data.charts` and populate `charts` with the relevant information.

Reply:
Thanks, I will do that